README.md

```diff
@@ -15,7 +29,7 @@ This page is available as an easy-to-read website at [https://ebookfoundation.gi
 
 This list was originally a clone of [StackOverflow - List of Freely Available Programming Books](https://web.archive.org/web/20140606191453/http://stackoverflow.com/questions/194812/list-of-freely-available-programming-books/392926) with contributions from Karan Bhangui and George Stocker.
 
-The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 221,000+ stars, 6,900+ commits, 1,900+ contributors, and 47,100+ forks.
+The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 221,000+ stars, about 9,300 watchers, up to 6,900 commits, 1,900+ contributors, and 47,100+ forks.
```

Comment:
```suggestion
The list was moved to GitHub by Victor Felder for collaborative updating and maintenance. It has grown to become one of [GitHub's most popular repositories](https://octoverse.github.com/), with 226,000+ stars, about 9,600 watchers, more than 7,000+ commits, 1,900+ contributors, and 47,700+ forks.
```

Reply:
`up to`, to me, sounds uncertain to me, like we don't know how many commits there are, more than suits better IMO

Reply:
> `up to`, to me, sounds uncertain to me, like we don't know how many commits there are, `more than` suits better IMO

And what about `near` or `approximately`?

Reply:
There needs to be the certainty, otherwise it sounds like a guess, also since the number isn't declining, `more than` suits better since the number will grow over time,